control/excalibur/client/client.py

```diff
@@ -67,11 +67,9 @@ class ExcaliburDefinitions(object):
     FEM_LFSR_BYPASS_MODE_ENABLED = 1 
     FEM_LFSR_BYPASS_MODE_NAMES = ('disabled', 'enabled')
     
-    FEM_COUNTER_DEPTH_MAP = OrderedDict(sorted(
-        {'1': 0, '6': 1, '12': 2, '24': 3, 'dual12': 4}.items(), 
-        key=lambda t: t[1]
-    ))
-    FEM_COUNTER_DEPTH_NAMES = FEM_COUNTER_DEPTH_MAP.keys()
+    FEM_COUNTER_DEPTH_MAP = OrderedDict(
+        (('1', 0), ('6', 1), ('12', 2), ('24', 3), ('dual12', 4))
+    )   FEM_COUNTER_DEPTH_NAMES = FEM_COUNTER_DEPTH_MAP.keys()
```

Comment:
This is not valid. I guess this isn't checked in the Travis build?

Reply:
```suggestion
    FEM_COUNTER_DEPTH_MAP = OrderedDict(
        (('1', 0), ('6', 1), ('12', 2), ('24', 3), ('dual12', 4))
    )
    FEM_COUNTER_DEPTH_NAMES = FEM_COUNTER_DEPTH_MAP.keys()
```

Reply:
It's trying to use the suggested change feature that messed this up. One minute I'll push the correct one.